src/createLambdaProxyContext.js

```diff
@@ -78,14 +78,14 @@ module.exports = function createLambdaProxyContext(request, options, stageVariab
       stage: options.stage,
       requestId: `offlineContext_requestId_${utils.randomId()}`,
       identity: {
-        cognitoIdentityPoolId: 'offlineContext_cognitoIdentityPoolId',
-        accountId: 'offlineContext_accountId',
-        cognitoIdentityId: request.headers['cognito-identity-id'] || 'offlineContext_cognitoIdentityId',
-        caller: 'offlineContext_caller',
-        apiKey: 'offlineContext_apiKey',
+        cognitoIdentityPoolId: process.env.SLS_COGNITO_IDENTITY_POOL_ID || 'offlineContext_cognitoIdentityPoolId',
+        accountId: process.env.SLS_ACCOUNT_ID || 'offlineContext_accountId',
+        cognitoIdentityId: request.headers['cognito-identity-id'] || process.env.SLS_COGNITO_IDENTITY_ID || 'offlineContext_cognitoIdentityId',
```

Comment:
The missing parameter should go here

Reply:
I think there were not any params missing. It was just some wrong text in the docs